src/seedu/addressbook/Main.java

```diff
@@ -84,7 +86,12 @@ private void runCommandLoopUntilExitCommand() {
         do {
             String userCommandText = ui.getUserCommand();
             command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
+            CommandResult result = null;
+            try {
+                result = executeCommand(command);
+            } catch (StorageOperationException e) {
+                e.printStackTrace();
+            }
             recordResult(result);
```

Comment:
Remember to check for what happens if `result=null` before trying to `recordResult()`. Alternatively, you can initialize result as `CommandResult result = new CommandResult("")`  to prevent `recordResult()` from throwing a `NullPointerException`.